.github/workflows/release.yml

```diff
@@ -81,3 +81,31 @@ jobs:
               run: |
                 make POEDITOR_API_ACCESS="${POEDITOR_API_ACCESS}" POEDITOR_PROJECT_ID="306433" build
                 twine upload dist/*
+    docker:
+      name: Push to DockerHub
+      runs-on: ubuntu-latest
+      steps:
+        -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```

Comment:
Is this really needed?

Reply:
I'm not entirely sure, to be honest. I was going off of what was at the top of the page: "v2 of this action includes significant updates and now uses Docker Buildx. It works with 3 new actions (login, setup-buildx and setup-qemu) that we have created".

Reply:
Oh, I see, OK for me, then!